lib/multi_patient_api/bulk_export_validation_tester.rb

```diff
@@ -0,0 +1,163 @@
+require_relative './profile_guesser'
+
+module BulkExportValidationTester
+  include USCore::MustSupportTest
+  include ProfileGuesser
+
+  MAX_NUM_COLLECTED_LINES = 100
+  MIN_RESOURCE_COUNT = 2
+
+  def observation_profiles
```

Comment:
These seem like metadata rather than profiles.

Reply:
I'm with you but naming gets complicated because I need to leave `metadata` available for the calls to it within `perform_must_support_tests`. I'll try to get more descriptive.